{{cookiecutter.name}}/.golangci.yml

```diff
@@ -5,25 +5,37 @@ linters:
   enable:
     - megacheck
     - godox
-    - interfacer
     - goconst
     - whitespace
     - stylecheck
-    - golint
+    - revive
     - misspell
     - depguard
+  disable:
+    - scopelint
+    - maligned
   presets:
-    - bugs        # bodyclose, errcheck, gosec, govet, rowserrcheck, scopelint, staticcheck, typecheck
-    - complexity  # gocognit, gocyclo, nakedret
-    - format      # gofmt, goimports
-    - performance # bodyclose, maligned, prealloc, rowserrcheck
-    - unused      # deadcode, ineffassign, structcheck, unparam, unused, varcheck
+    - bugs # asciicheck, bidichk, bodyclose, contextcheck, durationcheck, errcheck, errchkjson, errorlint, exhaustive, exportloopref, gosec, govet, makezero, nilerr, noctx, rowserrcheck, scopelint, sqlclosecheck, staticcheck, typecheck
+    - complexity # cyclop, funlen, gocognit, gocyclo, maintidx, nestif
+    - format # decorder, gci, gofmt, gofumpt, goimports
+    - performance # bodyclose, maligned, noctx, prealloc
+    - unused # deadcode, ineffassign, structcheck, unparam, unused, varcheck
   fast: false
 
 # configuration for specific linters
 linters-settings:
+  funlen:
+    lines: 80
+  cyclop:
+    max-complexity: 12
+    skip-tests: true
   nakedret:
     max-func-lines: 0 # no naked returns
+  gci:
+    sections:
+      - standard # Captures all standard packages if they do not match another section.
+      - default # Contains all imports that could not be matched to another section type.
+      - prefix(go.soon.build/cms/workflow) # Groups all imports with the specified Prefix.
```

Comment:
This prefix would need to come from the go module variable. I wasn't sure how I felt about `gci` - it seems more flexible in terms of config over goimports but it doesn't support autofix so it can get quite annoying to manually sort the imports.

Reply:
Ahh yeh sorry copy and paste error. Yes I was debating it too for the same reasons. We aren't really using any of the flexibility that it offers so happy to disable it and stick with goimports

Reply:
Looks like they're bringing back autofix support [here](https://github.com/golangci/golangci-lint/pull/2892) but it's not released yet in golangci-lint so might be worth disabling for now and then we can bring it back once that is released.